userspace/falco/app/actions/helpers_interesting_sets.cpp

```diff
@@ -75,15 +88,51 @@ static void select_event_set(falco::app::state& s, const libsinsp::events::set<p
 	auto base_event_set = libsinsp::events::sinsp_state_event_set();
 	s.selected_event_set = rules_event_set.merge(base_event_set);
 
+	auto user_base_syscalls_names = s.config->m_base_syscalls;
+	if (!user_base_syscalls_names.empty())
+	{
+		auto valid_events = libsinsp::events::names_to_event_set(user_base_syscalls_names);
+		auto negative_names = extract_negative_base_syscalls_names(user_base_syscalls_names);
+		auto valid_negative_events = libsinsp::events::names_to_event_set(negative_names);
+
+		auto n_invalid_positive_names = (user_base_syscalls_names.size() - negative_names.size()) - libsinsp::events::names_to_sc_set(user_base_syscalls_names).size();
+		if (n_invalid_positive_names > 0)
+		{
+			std::cerr << "User config base_syscalls includes ("
```

Comment:
We discussed to only emit light warnings if there is invalid user input, but there will be no fall-back. This option is 100% override.